packages/perseus-core/src/widgets/numeric-input/util.ts

```diff
@@ -0,0 +1,64 @@
+import type {
+    PerseusNumericInputAnswerForm,
+    PerseusNumericInputWidgetOptions,
+} from "../../data-schema";
+
+/**
+ * uniqueAnswerForms takes a list of answer forms and returns a list of unique
+ * answer forms. This is useful for ensuring that we don't show duplicate examples
+ * to the user.
+ */
+export const getUniqueAnswerForms = function (
+    list: readonly PerseusNumericInputAnswerForm[],
+): PerseusNumericInputAnswerForm[] {
+    // We use a Set to keep track of the forms we've already seen.
+    const foundForms = new Set<string>();
+    return list.filter((form) => {
+        // If we've already seen this form, skip it.
+        if (foundForms.has(form.name)) {
+            return false;
+        }
+        // Otherwise, add it to the set and return true.
+        foundForms.add(form.name);
+        return true;
```

Comment:
Is it correct to not take `simplify` into account when de-duplicating answer forms? E.g. if we have

```ts
const answerForms = [
  {name: "proper", simplify: "required"},
  {name: "proper", simplify: "optional"},
];
```

Only `{name: "proper", simplify: "required"}` will be present in the output. Whether `simplify` is `"required"` or `"optional"` in the output depends on the order in which the answers were originally listed, which is awkward, since I don't think the order is otherwise important.

Reply:
I noticed there's a very similar function, `unionAnswerForms`, in `numeric-input.tsx`. I think yours is better (better-typed, easier to read, possibly faster) but the two functions should probably be combined if they represent the same concept.